Dynamo.Stream.Handler/src/Dynamo.Stream.Handler/Migrations/20240117115914_AddViewsForBi.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Dynamo.Stream.Handler.Migrations
+{
+    public partial class AddViewsForBi : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // SQL command to create AnonymisedParticipantView
+            migrationBuilder.Sql(@"
+                CREATE VIEW Participants_Anonymised AS
+                SELECT p.Id,
```

Comment:
Minor, but useful when comparing results. Keep the ordering of the columns (or their anonymised equivalents) the same between table and view.

```
    `participants`.`Id`,
    `participants`.`FirstName`,
    `participants`.`LastName`,
    `participants`.`RegistrationConsent`,
    `participants`.`RegistrationConsentAtUtc`,
    `participants`.`CreatedAt`,
    `participants`.`UpdatedAt`,
    `participants`.`Email`,
    `participants`.`EthnicBackground`,
    `participants`.`EthnicGroup`,
    `participants`.`DateOfBirth`,
    `participants`.`RemovalOfConsentRegistrationAtUtc`,
    `participants`.`HasLongTermCondition`,
    `participants`.`GenderIsSameAsSexRegisteredAtBirth`,
    `participants`.`MobileNumber`,
    `participants`.`LandlineNumber`,
    `participants`.`NHSNumber`,
    `participants`.`IsDeleted`,
    `participants`.`DailyLifeImpactId`,
    `participants`.`CommunicationLanguageId`,
    `participants`.`GenderId`
```